deb/list.go

```diff
@@ -266,6 +266,19 @@ func (l *PackageList) Strings() []string {
 	return result
 }
 
+// FullNames builds a list of package {name}_{version}_{arch}
+func (l *PackageList) FullNames() []string {
+	result := make([]string, l.Len())
+	i := 0
+
+	for _, p := range l.packages {
```

Comment:
You don't need the `i` variable above (and the `i++` increment below). Just use the index from the `range` here.

Reply:
This does need the `i`/`i++` variable because `l.packages` is type `map[string]*Package` and so the index is the map key string.

Reply:
Oh, right!